openwisp_controller/connection/static/connection/css/batch-command.css

```
@@ -0,0 +1,252 @@
+  color: var(--error-fg);
+}
+.command-status.in-progress {
+  color: var(--body-quiet-color);
+}
+.command-status.skipped {
+  color: var(--body-quiet-color);
+  opacity: 0.7;
+}
+.command-output pre {
+  white-space: pre-wrap;
+  word-wrap: break-word;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)